website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -44,6 +44,8 @@ For example, you would use the following command if you use Snowflake:
 pip install dbt-core dbt-snowflake
 ```
 
+For the time being, we have maintained install-time dependencies to avoid breaking existing scripts in surprising ways; `pip install dbt-snowflake` will continue to install `dbt-core` implicitly. Given that we may remove this implicit dependency in future versions, we strongly encourage you to update install scripts **now**.
```

Comment:
```suggestion
For the time being, we have maintained install-time dependencies to avoid breaking existing scripts in surprising ways; `pip install dbt-snowflake` will continue to install the latest versions of both `dbt-core` and 1dbt-snowflake`. Given that we may remove this implicit dependency in future versions, we strongly encourage you to update install scripts **now**.
```

Reply:
opened up a pr https://github.com/dbt-labs/docs.getdbt.com/pull/5428 to address the 1dbt-snowflake typo.